packages/features/settings/layouts/SettingsLayout.tsx

```diff
@@ -135,41 +135,59 @@ tabs.find((tab) => {
 // The following keys are assigned to admin only
 const adminRequiredKeys = ["admin"];
 const organizationRequiredKeys = ["organization"];
+const organizationAdminKeys = ["privacy", "appearance", "billing", "OAuth Clients", "SSO", "directory_sync"];
```

Comment:
<img width="1512" alt="Screenshot 2024-04-19 at 3 40 23 PM" src="https://github.com/calcom/cal.com/assets/53316345/fe29eef5-6e8c-4ecd-9d96-6302a4902e26">

even 'members' tab should be hidden

Reply:
This should only be hidden when an org is private - Will do this in a follow up PR as it requries fetching this info from somwhere and im trying to get around adding another fetch in here for a loading state